src/Nethermind/Nethermind.Core/Encoding/RlpStream.cs

```diff
@@ -212,6 +212,11 @@ public void Encode(byte value)
             }
         }
 
+        public void Encode(bool value)
+        {
+            Encode(value ? (byte) 1 : (byte) 0);
```

Comment:
there is some strange place in eth where this bool encoding looks different...

Reply:
its same 0 byte will be encoded as 128